src/main/java/org/bihealth/mi/easysmpc/components/DialogEmailConfig.java

```diff
@@ -232,11 +376,36 @@ public ConnectionIMAPSettings showDialog(){
      */
     @Override
     public void stateChanged(ChangeEvent e) {
```

Comment:
Is it really necessary that this class is a change listener in another component? Would prefer to have the other component know about this class and explicitly call "switch" or something similar from the outside.

Reply:
Changed as requested